src/baklava.ts

```diff
@@ -1,40 +1,41 @@
+export { default as BlLink } from "./components/link/bl-link";
+export { default as BlNotification } from "./components/notification/bl-notification";
+export { default as BlNotificationCard } from "./components/notification/card/bl-notification-card";
 export { default as BlPagination } from "./components/pagination/bl-pagination";
+export { default as BlPopover } from "./components/popover/bl-popover";
 export { default as BlProgressIndicator } from "./components/progress-indicator/bl-progress-indicator";
 export { default as BlRadioGroup } from "./components/radio-group/bl-radio-group";
 export { default as BlRadio } from "./components/radio-group/radio/bl-radio";
 export { default as BlSelect } from "./components/select/bl-select";
 export { default as BlSelectOption } from "./components/select/option/bl-select-option";
-export { default as BlTab } from "./components/tab-group/tab/bl-tab";
+export { default as BlSpinner } from "./components/spinner/bl-spinner";
+export { default as BlSplitButton } from "./components/split-button/bl-split-button";
+export { default as BlSwitch } from "./components/switch/bl-switch";
 export { default as BlTabGroup } from "./components/tab-group/bl-tab-group";
 export { default as BlTabPanel } from "./components/tab-group/tab-panel/bl-tab-panel";
-export { default as BlTextarea } from "./components/textarea/bl-textarea";
-export { default as BlTooltip } from "./components/tooltip/bl-tooltip";
-export { default as BlPopover } from "./components/popover/bl-popover";
-export { default as BlDropdown } from "./components/dropdown/bl-dropdown";
-export { default as BlDropdownItem } from "./components/dropdown/item/bl-dropdown-item";
-export { default as BlDropdownGroup } from "./components/dropdown/group/bl-dropdown-group";
-export { default as BlSwitch } from "./components/switch/bl-switch";
-export { default as BlSpinner } from "./components/spinner/bl-spinner";
-export { default as BlNotification } from "./components/notification/bl-notification";
-export { default as BlNotificationCard } from "./components/notification/card/bl-notification-card";
+export { default as BlTab } from "./components/tab-group/tab/bl-tab";
 export { default as BlTable } from "./components/table/bl-table";
-export { default as BlTableHeader } from "./components/table/table-header/bl-table-header";
 export { default as BlTableBody } from "./components/table/table-body/bl-table-body";
-export { default as BlTableRow } from "./components/table/table-row/bl-table-row";
-export { default as BlTableHeaderCell } from "./components/table/table-header-cell/bl-table-header-cell";
 export { default as BlTableCell } from "./components/table/table-cell/bl-table-cell";
-export { default as BlSplitButton } from "./components/split-button/bl-split-button";
-export { default as BlCalendar } from "./components/calendar/bl-calendar";
-export { default as BlDatePicker } from "./components/datepicker/bl-datepicker";
```

Comment:
why did we change casing? that could be a breaking change

Reply:
It's breaking tests and btw it already named like that
`export default class BlDatepicker extends DatepickerCalendarMixin {`

Reply:
We should talk with @dilandoogan